lib/member_holding.rb

```diff
@@ -52,7 +52,10 @@ def parse(str)
   # rubocop:disable Metrics/MethodLength
   def set(col_type, col_val)
     if col_val.nil?
-      raise ColValError, "col_val for col_type #{col_type} is nil"
+      Services.scrub_logger.warn(
```

Comment:
Maybe worth a test that it warns but doesn't raise?

Reply:
Sure.

Reply:
Added https://github.com/hathitrust/holdings-backend/pull/92/commits/579c3cf38d7de7dd0518ffbf97e9e140f897ce23